tests/karma/unit/services/enketo-prepopulation-data.js

```diff
@@ -35,10 +35,12 @@ describe('EnketoPrepopulationData service', function() {
             '<user>' +
               '<name/>' +
             '</user>' +
-            '<location>' +
-              '<lat/>' +
-              '<long/>' +
-            '</location>' +
+            '<meta>' +
+              '<location>' +
+                '<lat/>' +
+                '<long/>' +
+              '</location>' +
+            '</meta>' +
```

Comment:
I added `<meta />` because AFAICT the structure is `inputs->meta->location`, so I'm wondering if the test data is just wrong? If you don't think this is true we can change the code to pass a `childMatcher` that allows for either.